assets/arrival.js

```diff
@@ -131,6 +132,7 @@ class ArrivalTimes extends Component {
         if (stop) {
           const [lng, lat, name] = stop;
           document.title = `Bus arrival times for ${code + ' - ' + name}`;
+          document.querySelectorAll('[name="apple-mobile-web-app-title"]')[0].setAttribute("content", name);
```

Comment:
Could use `document.querySelector` instead which returns the first element instead of an array of elements. Also `"content"` should be single-quoted.